commit 82dbdd3b80734b10a1443cff5e246b15cd011d39
Author: John Doe <jdoe@email.com>
Date:   Sun Feb 11 23:46:15 2024 +0800

    Remove final use of legacy SkFontMgr methods    2023-12-15 02:11:11    fa624b96d056ed5b498dc8f79a9e32c3b7917134

diff --git a/skia/BUILD.gn b/skia/BUILD.gn
index a06504d375c4e..af458f346a021 100644
--- a/skia/BUILD.gn
+++ b/skia/BUILD.gn
@@ -84,8 +84,6 @@ config("skia_config") {
     "SK_DISABLE_LEGACY_DEFAULT_TYPEFACE",
     "SK_DISABLE_LEGACY_VULKAN_BACKENDSEMAPHORE",
     "SK_DISABLE_LEGACY_CREATE_CHARACTERIZATION",
-    "SK_DISABLE_LEGACY_FONTMGR_REFDEFAULT",
-    "SK_DISABLE_LEGACY_FONTMGR_FACTORY",
   ]
 
   include_dirs = [
